internal/npm_install/test/golden/rxjs/BUILD.bazel.golden

```diff
@@ -13,7 +13,6 @@ filegroup(
         "//:node_modules/rxjs/InnerSubscriber.d.ts",
         "//:node_modules/rxjs/InnerSubscriber.js",
         "//:node_modules/rxjs/InnerSubscriber.js.map",
-        "//:node_modules/rxjs/LICENSE.txt",
```

Comment:
I guess it's technically a breaking change if users referred to one of these target directly?

Reply:
Nope. No breaking change here. This is just testing that  the `include_files` works are intended and that any files that aren't included still end up the __all_files target.